sim/paladin/items.go

```diff
@@ -0,0 +1,92 @@
+package paladin
+
+import (
+	"github.com/wowsims/cata/sim/core"
+	"github.com/wowsims/cata/sim/core/stats"
+	"time"
+)
+
+// Tier 11 ret
+var ItemSetReinforcedSapphiriumBattleplate = core.NewItemSet(core.ItemSet{
+	Name: "Reinforced Sapphirium Battleplate",
+	Bonuses: map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			paladin := agent.(PaladinAgent).GetPaladin()
+
+			paladin.AddStaticMod(core.SpellModConfig{
+				Kind:       core.SpellMod_DamageDone_Flat,
+				ClassMask:  SpellMaskTemplarsVerdict,
+				FloatValue: 0.1,
+			})
+		},
+		4: func(agent core.Agent) {
+			// Handled in inquisition.go
+		},
+	},
+})
+
+// PvP sets
+func (paladin *Paladin) registerPvpSets() {
+	var setNames = []string{
+		"Gladiator's Vindication",
+		"Bloodthirsty Gladiator's Vindication",
+		"Vicious Gladiator's Vindication",
+		"Ruthless Gladiator's Vindication",
+		"Cataclysmic Gladiator's Vindication",
+	}
+
+	pvpSetBonuses := map[int32]core.ApplyEffect{
+		2: func(agent core.Agent) {
+			paladin := agent.(PaladinAgent).GetPaladin()
+
+			paladin.AddStat(stats.Strength, 70)
+		},
+		4: func(agent core.Agent) {
+			paladin := agent.(PaladinAgent).GetPaladin()
+
+			paladin.AddStat(stats.Strength, 90)
+			paladin.AddStaticMod(core.SpellModConfig{
+				Kind:      core.SpellMod_Cooldown_Flat,
+				ClassMask: SpellMaskJudgement,
+				TimeValue: -1 * time.Second,
+			})
+		},
+	}
+
+	for _, setName := range setNames {
+		core.NewItemSet(core.ItemSet{
+			Name:    setName,
+			Bonuses: pvpSetBonuses,
+		})
+	}
```

Comment:
Not the right way to do this. In Wow PVP sets are shared as in you can have different sets but still activate the bonuses. This wont have the same effect. We have a mechanism for adding alternative set names and we will extend it to take more then 2 (current limit). For now you can just remove this and not implement Pvp sets until we have the core functionality ready for this